docs/source/user_guide/user_guide/parallelization.md

```diff
@@ -0,0 +1,11 @@
+# Parallelization
+
+Each top-level for-loop will be parallelized, within a kernel
+- adding a non-static `if` over the top of a for-loop will lead to the for-loop not being parallelized
+- a for-loop inside one or more inlined @ti.func, and/or static `if` will be still be parallelized however
+
+Each top-level for-loop will be launched as a separate GPU kernel, under the hood.
```

Comment:
Can you clarify that it implies? It is better to put everything under the same for-loop if possible? Or to split each line in separate for-loop? Does it change anything?

Reply:
clarify in the doc? sure, ok.

Reply:
added a bunch more stuff. (not commited/pushed yet)